js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -33,39 +34,57 @@
 	export let applyInputSample: (sample: TWidgetExample, opts?: ExampleRunOpts) => void = () => {};
 	export let validateExample: (sample: WidgetExample) => sample is TWidgetExample;
 	export let exampleQueryParams: QueryParam[] = [];
+	export let isDisabled = false;
```

Comment:
I think it gets overridden directly by the line below no?
```suggestion
```

Reply:
Ah after re-reading I'm thinking this should probably not be exported
```suggestion
	let isDisabled = false;
```